Source/Applications/SystemCenterNotification/Scripts/TSX/EmailTypes/DatasourceUI/DataSourceTesting.tsx

```diff
@@ -34,31 +34,57 @@ import EventSelect from '../TriggerUI/EventSelect';
 import * as $ from 'jquery';
 import { Application } from '@gpa-gemstone/application-typings';
 import { pd } from 'pretty-data';
+import Table from '@gpa-gemstone/react-table';
 
 declare var homePath;
 declare var version;
 
+// #ToDO move to gpa-gemstone
+const CircledX = <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
+    fill="currentColor" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round"
+    className="feather feather-file-text">
+        <circle stroke="rgb(139, 0, 0)" fill={'none'} cx="12" cy="12" r="11"/>
+        <path stroke="rgb(139, 0, 0)" strokeWidth={3} d="M 6 6 L 18 18" />
+        <path stroke="rgb(139, 0, 0)" strokeWidth={3} d="M 18 6 L 6 18" />
+    </svg>
+
+const CircleCheck = <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24"
+    viewBox="0 0 24 24" fill="currentColor" stroke="currentColor" strokeWidth="2"
+    strokeLinecap="round" strokeLinejoin="round" className="feather feather-file-text">
+        <circle stroke="rgb(0, 100, 0)" fill={'none'} cx="12" cy="12" r="11" />
+    <path stroke="rgb(0, 100, 0)" fill={'none'}  strokeWidth={3} d="M 5 15 L 10 20 L 18 5"/>
+    </svg>
```

Comment:
```
// #ToDO move to gpa-gemstone
const CircledX = <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"
    fill="currentColor" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round"
    className="feather feather-file-text">
        <circle stroke="rgb(139, 0, 0)" fill={'none'} cx="12" cy="12" r="11"/>
        <path stroke="rgb(139, 0, 0)" strokeWidth={3} d="M 6 6 L 18 18" />
        <path stroke="rgb(139, 0, 0)" strokeWidth={3} d="M 18 6 L 6 18" />
    </svg>

const CircleCheck = <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24"
    viewBox="0 0 24 24" fill="currentColor" stroke="currentColor" strokeWidth="2"
    strokeLinecap="round" strokeLinejoin="round" className="feather feather-file-text">
        <circle stroke="rgb(0, 100, 0)" fill={'none'} cx="12" cy="12" r="11" />
    <path stroke="rgb(0, 100, 0)" fill={'none'}  strokeWidth={3} d="M 5 15 L 10 20 L 18 5"/>
    </svg>
```
Do these need to be moved before merging?

Reply:
No We don't want to wait to get this in until we get around to republishing `gpa-gemstone`